src/simulation/m_global_parameters.fpp

```diff
@@ -310,6 +318,7 @@ module m_global_parameters
     !! The finite-difference number is given by MAX(1, fd_order/2). Essentially,
     !! it is a measure of the half-size of the finite-difference stencil for the
     !! selected order of accuracy.
+    !$acc declare create(fd_order,fd_number)
```

Comment:
i'm curious why this is here if it wasn't in the code before?

Reply:
Ben and Maurop say this is a result of the mixup between what happens on device and the host. Someone just needs to sit down in a less busy space and iron it out.